compiler/il/OMRNode.cpp

```diff
@@ -388,6 +388,9 @@ OMR::Node::copyValidProperties(TR::Node *fromNode, TR::Node *toNode)
    UnionPropertyA_Type fromUnionPropertyA_Type = fromNode->getUnionPropertyA_Type();
    UnionPropertyA_Type toUnionPropertyA_Type = toNode->getUnionPropertyA_Type();
 
+   // copy all chidlren in case the node does not have a data type and it has to be computed from its children
```

Comment:
sp:   children

Reply:
Why not merge this comment into the previous comment which is already talking about special handling for typeless nodes?  This copyChildren requirement is just part of that.

Reply:
Fixed both.